app/views/hyrax/users/_left_sidebar.html.erb

```diff
@@ -1,7 +1,2 @@
-<%= image_tag @user.avatar.url(:thumb), width: 100 %>
-      <h3><%= @user.name %> </h3>
```

Comment:
when you remove this line - does the user name still appears on the profile page?